package.json

```diff
@@ -25,7 +25,7 @@
   "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "dependencies": {
-    "cosmiconfig": "^7.0.0",
+    "cosmiconfig": "^8.1.3",
```

Comment:
cosmiconfig 8 drops support for node 10. I'm looking at `engines` lower in this file and it seems like `babel-plugin-macros` still supports node 10, which would mean that it would have to drop that support in order to adopt cosmiconfig 8. I believe that dropping support for older node versions is something that will need to happen in a new major version, as it did for cosmiconfig.